### PR TITLE
PAYARA-2701 It is not possible to target a JDBC resource to a deployment group

### PR DESCRIPTION
**Pull request by MeroRai**

### CODE REVIEW

**Comment by MeroRai** [comment]
Jenkins test please

**Comment by MeroRai** [comment]
I mixed two PRs together. I will close the other PR, this PR should be safe to merge though.

File: nucleus/admin/util/src/main/java/org/glassfish/admin/cli/resources/CreateResourceRef.java [hunk 1]
```diff
@@ -184,9 +184,11 @@ public void execute(AdminCommandContext context) {
             }
             // create new ResourceRef for all instances of DG, if it's a dg
             if (refContainer instanceof DeploymentGroup && isElegibleResource(refName)) {
-                DeploymentGroup dg = (DeploymentGroup)refContainer;
+                DeploymentGroup dg = (DeploymentGroup) refContainer;
```
Comment (arjantijms) [inline nucleus/admin/util/src/main/java/org/glassfish/admin/cli/resources/CreateResourceRef.java]: Avoid names like "dg" and "svr"

**Comment by wcardenas** [comment]
Is this fixed in 5.183? I'm running 5.183 and still can't target a deployment group:

D:\>D:\ntst\payara\payara5.183\glassfish\bin\asadmin -p 51048 create-custom-resource --restype java.lang.String --factoryclass org.glassfish.resources.custom.factory.PrimitivesAndStringFactory --target CareManager-UI foo
remote failure: Target CareManager-UI is not a supported type. Command create-custom-resource supports these types of targets only: Stand alone instance, Default server, Domain, Cluster,
Command create-custom-resource failed.

**Comment by wcardenas** [comment]
So I just realized that this issue is related to JDBC resource. Is there also a fix for JNDI resources?

**Pull request merged**